std_msgs/Header header
geometry_msgs/Point left_eye_position
geometry_msgs/Point right_eye_position